examples/client-secure.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 The Cockroach Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cockroachdb-client-secure
+  labels:
+    app: cockroachdb-client
+spec:
+  serviceAccountName: cockroach-operator-default
+  initContainers:
```

Comment:
You do not need the init-container here.  You can just mount the cert that we create or create another cert and then mount it to the pod.

Reply:
> You can just mount the cert that we create or create another cert and then mount it to the pod.

Can you provide more context on how exactly this is done?

Reply:
Look at how the pods mount the certificates in the stateful set that is deployed by the operator. You can do the exact same thing. Also “init-container”. Autocorrect messed me up.

Reply:
@chrislovecnm Can you please point me to the relevant block? I can't find it in the repo. Our own [StatefulSet configs](https://github.com/cockroachdb/cockroach/blob/master/cloud/kubernetes/cockroachdb-statefulset-secure.yaml) use `initContainers` which I replicated here.